mon-pix/app/authenticators/gar.js

```diff
@@ -13,4 +14,13 @@ export default class GarAuthenticator extends BaseAuthenticator {
       source,
     });
   }
+
+  restore(data) {
+    return new RSVP.Promise((resolve, reject) => {
```

Comment:
L'utilisation de RSVP n'est plus utile dans le code de Pix compte tenu des navigateurs avec lesquels Pix doit être compatible. Mais gardons ce code et faisons une autre PR pour supprimer partout l'utilisation de RSVP.

```suggestion
    return new Promise((resolve, reject) => {
```

cf. https://github.com/1024pix/pix/blob/dev/docs/adr/0052-compatibilite-navigateurs.md
cf. https://caniuse.com/?search=Promise